### PR TITLE
Separate testing notebooks from source code

### PR DESCRIPTION
**Pull request by Vicbi**
# *Separate testing notebooks from source code*

## :recycle: Current situation & Problem
This pull request refactors the current project structure by separating the testing notebooks from the source code. This refactoring aims to streamline the repository by isolating the source code, enhancing maintainability, and improving focus within the repository.

## :gear: Release Notes 
- Refactored project by moving notebooks out of the source code repository to maintain a clean separation of concerns.
- Improved maintainability.

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ SPDX-License-Identifier: MIT
...
 
-# Spezi Data Pipeline Template
+# Spezi Data Pipeline
 
-[![Build and Test](https://github.com/StanfordSpezi/SpeziDataPipelineTemplate/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordSpezi/SpeziDataPipelineTemplate/actions/workflows/build-and-test.yml)
-[![codecov](https://codecov.io/gh/StanfordSpezi/SpeziDataPipelineTemplate/branch/main/graph/badge.svg)](https://codecov.io/gh/StanfordSpezi/SpeziDataPipelineTemplate)
-<a target="_blank" href="https://colab.research.google.com/github/StanfordSpezi/SpeziDataPipelineTemplate/blob/main/SpeziDataPipelineTemplate.ipynb">
+[![Build and Test](https://github.com/StanfordSpezi/SpeziDataPipeline/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordSpezi/SpeziDataPipeline/actions/workflows/build-and-test.yml)
+[![codecov](https://codecov.io/gh/StanfordSpezi/SpeziDataPipeline/branch/main/graph/badge.svg)](https://codecov.io/gh/StanfordSpezi/SpeziDataPipeline)
+<a target="_blank" href="https://colab.research.google.com/github/StanfordSpezi/SpeziDataPipeline/blob/main/SpeziDataPipeline.ipynb">
```
Comment (PSchmiedmayer) [inline README.md:15]: I don't think that we have a notebook in this repo anymore; we might want to link to the other template repo here or just remove the Colab badge here.

**Review by PSchmiedmayer — APPROVED** [review]
Thank you for removing the notebooks here!

I fixed the codecov errors by re-syncing the app with the GitHub organization.

Only had one minor comment, apart from this it looks great!

File: README.md (hunk 1 above)
Comment (Vicbi) [reply README.md:15]: You're right, we don't need it here. I have included it in the other repo!

Comment (Vicbi) [reply README.md:15]: Thanks for the prompt response and fir resolving the codecov issue! 🚀

**Pull request merged**